starfish/test/dataset_fixtures.py

```diff
@@ -123,7 +127,7 @@ def loaded_codebook(simple_codebook_json):
 
 @pytest.fixture(scope='function')
 def euclidean_decoded_intensities(small_intensity_table, loaded_codebook):
-    decoded_intensities = loaded_codebook.decode_euclidean(small_intensity_table)
+    decoded_intensities = loaded_codebook.decode_euclidean(small_intensity_table, max_distance=0)
```

Comment:
if max_distance==0, doesn't that mean everything is pruned from the results?  if so, can we assert that the decoded intensity table is empty?

Reply:
Close, it means all the matches need to be perfect, so we can't make that assertion.

Reply:
Ah!

Is it worth asserting on any of the resulting IntensityTables?   approx size?  i don't know what else is easily assertable...

Reply:
Added size assertion. 👍